examples/eth-with-802-1x.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: network-test
+  vars:
+    network_connections:
+      - name: eth0
+        type: ethernet
+        802.1x:
+          identity: myhost
+          private-key: /etc/pki/tls/client.key
+          # recommend vault encrypting the private key password
+          private-key-password: "p@55w0rD"
+          client-cert: /etc/pki/tls/client.pem
+          ca-cert:  /etc/pki/tls/cacert.pem
+
+  # certs have to be deployed first
+  pre_tasks:
```

Comment:
It is also possible to use `import_role` to have a task to run the role, removing the need for pre_tasks. This might be a better way of doing things to allow to easier re-use the example in another playbook. What do you think?

Example:
https://github.com/linux-system-roles/network/blob/master/tests/playbooks/tests_ethtool_features.yml#L77

Reply:
I can change this if you like. I tend to prefer to write playbooks with pre_tasks and post_tasks so all the roles are kept together but that is just a personal preference